deltachat-ios/DC/DcMsg.swift

```diff
@@ -283,8 +231,8 @@ public class DcMsg {
     }
 
     // DC_MSG_*
-    public var type: Int {
-        return Int(dc_msg_get_viewtype(messagePointer))
+    public var type: Int32 {
```

Comment:
What's the reason for `Int32` here? Why not just `Int` as before?

Reply:
before, type was casted on higher levels (see simplifications in this pr)

the reason "Int" is not working is that swift uses "Int32" for ffi-bindings